templates/list.html

```diff
@@ -6,6 +6,42 @@
      to experience if you were using someone else's built application?
  -->
 
-<div ng-controller="listController">
-  <h1>This is the list page!</h1>
+<div ng-controller="ListController">
+	<div class="container">
+		<h1>IMDB Movie List</h1>
+	</div>
+	<form ng-submit="getMovies()" class="form-inline">
+		<input type="text" class="form-control" id="movieSearchInput" placeholder="Enter Search Here" ng-model="movieText.title" required>
+		<button type="submit" class="btn btn-primary">submit</button>
+	</form>
+	<br>
+	<div class="nav">
+		<div class="container">
+			<ul class="pull-right">
+				<li>Sort by:</li>
+				<li ng-click="sorting = 'rank'">Rank 1-250</li>
+				<li ng-click="sorting = '-rank'">Rank 250-1</li>
+				<li ng-click="sorting = 'title'">Title A-Z</li>
+				<li ng-click="sorting = '-title'">Title Z-A</li>
+			</ul>
+		</div>
+	</div>
+	<br>
+	<div class="movie-list">
+		<div class="container">
```

Comment:
The layout for all the movies was excellent! You could add images next to the movie names and descriptions and it would look even better. You had the cleanest layout of all the submissions, I believe. But adding images would make the aesthetic even better! Glad you were thinking about it in terms of the design.